src/lib/controllers/ui-controller.js

```diff
@@ -1817,10 +1839,8 @@ export default class UIController {
   featureOptionChange (name, value) {
     let featureOptions = this.api.settings.getFeatureOptions() // eslint-disable-line prefer-const
     // TODO we need to refactor handling of boolean options
-    if (name === 'enableLemmaTranslations' ||
-        name === 'enableWordUsageExamples' ||
-        name === 'wordUsageExamplesMax' ||
-        name === 'wordUsageExamplesAuthMax') {
+    const nonTextFeatures = ['enableLemmaTranslations', 'enableWordUsageExamples', 'wordUsageExamplesMax', 'wordUsageExamplesAuthMax', 'enableMouseMove', 'mouseMoveDelay', 'mouseMoveAccuracy']
+    if (nonTextFeatures.indexOf(name) > -1) {
```

Comment:
Added handling for new settings

Reply:
Maybe it's better to use `includes()` here?